src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -185,14 +185,20 @@ namespace HeatBalanceSurfaceManager {
     // Data
     // MODULE PARAMETER DEFINITIONS:
     static std::string const BlankString;
+    Array1D<Real64> RefAirTemp; // inside surface convection reference air temperatures
```

Comment:
Put this here instead of in `DataHeatBalSurface` because `RefAirTemp` shows up as a local variables and/or array in many other places in the code - all going through the same gymnastics to figure out the air temp for inside surface convection. A definite code de-duplication opportunity.